README.md

```diff
@@ -188,6 +188,15 @@ $.Function :: Type
 
 Type comprising every Function value.
 
+#### `Integer`
+
+```haskell
+$.Integer :: Type
+```
+
+Type comprising every non-floating point [`ValidNumber`](#validnumber) value 
+between the range `-9007199254740992` and `9007199254740992`.
```

Comment:
I'll wait until #20 lands on master before that because I don't want to have to do the references at the bottom of the README twice.